docs/installation/configuration/cmake_options.rst

```diff
@@ -117,21 +111,52 @@ dependency on other options.
           more information on *Fast DDS* real-time configuration.
         - ``ON`` ``OFF``
         - ``OFF``
-    *   - :class:`INTERNAL_DEBUG`
-        - Activates |Log::Kind::Info| debug messages (See :ref:`dds_layer_log_intro`). |br|
-          For this option to have any effect, i.e. to print the information messages,  |br|
-          *Fast DDS* must be debug built. This is done by setting the :class:`CMAKE_BUILD_TYPE` |br|
-          option to `Debug`. Moreover, :class:`INTERNAL_DEBUG` is set to ``ON`` if |br|
-          :class:`EPROSIMA_BUILD` is ``ON``.
-        - ``ON`` ``OFF``
-        - ``OFF``
     *   - :class:`SQLITE3_SUPPORT`
         - Builds the |SQLITE3_PLUGIN|, which enables the |TRANSIENT_DURABILITY_QOS-api| |br|
           and |PERSISTENT_DURABILITY_QOS-api| options for the :ref:`durabilitykind` |br|
           and therefore the :ref:`persistence_service`.
         - ``ON`` ``OFF``
         - ``ON``
 
+Log options
+-----------
+*Fast DDS* uses its own configurable **Log module**  with different verbosity levels.
+Please, refer to :ref:`dds_layer_log_intro` section for more information.
+
+This module can be configured using *Fast DDS* CMake arguments regarding the following options.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Possible values
+        - Default
+    *   - :class:`LOG_CONSUMER_DEFAULT`
+        - Selects the default log consumer for the logging module. ``AUTO`` has the same behavior as ``STDOUT``. |br|
```

Comment:
Break the line to fit the table in the page.
```suggestion
        - Selects the default log consumer for the logging module. |br|
        ``AUTO`` has the same behavior as ``STDOUT``. |br|
```

Reply:
:+1: